.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: release
+
+on:
+  push:
+    branches:
+      - release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: latest
+          cache: "npm"
+
+      - name: Install dependencies
+        run: npm ci
```

Comment:
You may have to do `npm i` here. Not sure if we can run the tests without dev dependencies.

Reply:
Is there a way we can test this? I honestly don't have much experience with GitHub pipelines

Reply:
Yeah, it can be a little finicky. I typically will create and test them on a different branch by changing the top section and then when it's ready I modify it to the target branch